jbt-it/neomdb
# Added route for retrieving permission

Added a route for retrieving all permissions of a single member (similar to the `retrieveUserData` function in `authController.jsx`) with the specified id.

How to test:
1. Check if the member w.luft (8167) has permissions in the database
2. Make following request in an api client of your choice:
GET _https://localhost:3030/users/8167/permissions_
--> This should return the list of permissions of w.luft
3. Make following request in an api client of your choice:
GET _https://localhost:3030/users/9999/permissions_
--> This should return the message 'Member with id "9999" does not exist'
